metapackages/auto-instrumentations-node/package.json

```diff
@@ -23,6 +27,7 @@
     "prewatch": "npm run precompile",
     "prepare": "npm run compile",
     "tdd": "yarn test -- --watch-extensions ts --watch",
+    "pretest": "npm run compile",
```

Comment:
When running locally, I understand this script will run automatically when running: `npm run test`, but in ci, the compilation already happened in the "prepare" script, so why do we need this line for?

Reply:
This was added to make sure we use the latest changes every time we run `npm run test` as the test here depends on the compiled register.js. Will remove It to reduce cicd execution time.